renku/cli/workflow.py

```diff
@@ -149,7 +162,13 @@
 
 The set of possible values for a parameter can be given by ``--map`` command
 line argument or by specifying them in a values YAML file and passing that
-using ``--mapping``.
+using ``--mapping``. Content of the mapping file for the above example
+should be:
+
+.. code-block:: yaml
+
+parameter-1: [1,2,3]
+parameter-2: [10,20]
```

Comment:
For setting a value on a child workflow, would it be `child.parameter-1` or 
```
child:
  parameter-1
```
?

Reply:
yep. just as in case of `workflow execute`.

Reply:
Both work? nice

Reply:
in case of execute there's a test for it using both yaml and cli interface: https://github.com/SwissDataScienceCenter/renku-python/blob/develop/tests/cli/test_workflow.py#L517